go.mod

```diff
@@ -13,7 +13,7 @@ require (
 	github.com/docker/buildx v0.10.4
 	github.com/docker/cli v23.0.5+incompatible
 	github.com/docker/cli-docs-tool v0.5.1
-	github.com/docker/docker v23.0.5+incompatible
+	github.com/docker/docker v23.0.6+incompatible
```

Comment:
https://github.com/docker/compose/pull/10542#discussion_r1188432369


> We should probably update `docker/docker` as well, and if we do, update containerd, runc, and ttrpc (go modules won't find those indirect dependencies because docker/docker and docker/cli don't have a `go.mod`); https://github.com/moby/moby/compare/v23.0.5..v23.0.6